addons/hr_recruitment/views/hr_applicant_views.xml

```diff
@@ -226,6 +226,7 @@
                 <field string="Email" name="email_from" filter_domain="[('email_from', 'ilike', self)]"/>
                 <field name="job_id"/>
                 <field name="department_id" operator="child_of"/>
+                <field name="company_id"/>
```

Comment:
should be : 
`<field name="company_id" groups="base.group_multi_company"/>`
Since if we're not in a multi-company setup, we don't want the field to appear in the search